src/App.js

```diff
@@ -20,9 +20,9 @@ import {
 
 const auth = new Auth();
 
-const handleAuthentication = (nextState, replace) => {
+const handleAuthentication = (nextState, replace => {
```

Comment:
Typo in the function parameter list, should be a comma instead of '=>' after "replace".